src/components/WwwFrame/TheFooter.vue

```diff
@@ -144,6 +144,15 @@
 									Partner with us
 								</router-link>
 							</li>
+							<li>
+								<a
+									:href="employeeEngagementUrl"
+									v-kv-track-event="['Footer', 'click-Work with us-Employee engagement']"
```

Comment:
This is under the Get to know us section so the 2nd param here should be:
`click-Get to know us-Employee engagement`.

Please update in both locations.

Reply:
Revised and thanks for the find! :)